skimage/exposure/_adapthist.py

```diff
@@ -306,13 +287,14 @@ def map_histogram(hist, min_val, max_val, n_pixels):
 
 
 def interpolate(image, slices, maps, lut):
-    """Find the new grayscale level for a region using bilinear interpolation.
+    """Find the new grayscale level for a region
+    using multilinear interpolation.
 
     Parameters
     ----------
     image : ndarray
         Full image.
-    slices : list of array-like
+    slices : list of slices
```

Comment:
can't this variable be renamed to `slices_list` for clarity?

Reply:
Done.